### PR TITLE
Update CRD catalog docs

### LINKED ISSUE #365: Update the CRD catalog docs

### PR DESCRIPTION
**Pull request by sgayangi**
## Purpose
Fixes #365

### CODE REVIEW

**Review by Krishanx92 — APPROVED** [review]

File: en/docs/catalogs/samples/api-cr.md [hunk 1]
```diff
@@ -24,4 +26,4 @@ spec:
   organization: apk-org
 ```
 
-Refer [Manage API](../../create-api/create-api-overview/) for more information on how to configure API.
\ No newline at end of file
+Refer [Manage API](../../create-api/create-api-overview.md) for more information on how to configure API.
```
Comment (PasanT9) [inline en/docs/catalogs/samples/api-cr.md:29]: Shall we fix this to `APIs`

File: en/docs/catalogs/samples/api-policy.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
-# API Policy
+## API Policy Samples
 
-API policies are a powerful tool in WSO2 APK for enforcing business logic and modifying API requests and responses. You can attach policies at the gateway-level, API-level or operation-level, in either the request or response flow
+Refer [Manage API Policy](../../create-api/create-and-attach-api-policies/api-policies-overview.md) for more information on how to configure API Policy with API.
```
Comment (PasanT9) [inline en/docs/catalogs/samples/api-policy.md:3]: Shall we fix this to `an API Policy with an API`?

File: en/docs/catalogs/samples/api-cr.md (hunk 1 above)
Comment (sgayangi) [reply en/docs/catalogs/samples/api-cr.md:29]: Fixed in PR #583

File: en/docs/catalogs/samples/api-policy.md (hunk 2 above)
Comment (sgayangi) [reply en/docs/catalogs/samples/api-policy.md:3]: Fixed in PR #583

**Pull request merged**